server/views/pages/deleteReport.njk

```diff
@@ -0,0 +1,39 @@
+{% extends "../partials/layout.njk" %}
+{% from "../partials/breadCrumb.njk" import breadcrumb %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Delete report" %}
+
+{% block beforeContent %}
+  {{ breadcrumb() }}
+{% endblock %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+
+  <p class="govuk-body" data-qa="delete-report-p1">Draft adjudication report will be deleted, please confirm.</p>
```

Comment:
I would have a look at the screenshots and ACs provided in the ticket for this page -
https://dsdmoj.atlassian.net/browse/NN-4743

we need the prisoner data at the top (I sent you an example of this in slack the other day) and there should just be a form with a submit button and a cancel link

Reply:
It also doesn't look like you're rendering this .njk file anywhere?

Reply:
Done.